test/cpp/microbenchmarks/helpers.cc

```diff
@@ -37,11 +39,16 @@ void TrackCounters::AddToLabel(std::ostream &out, benchmark::State &state) {
     out << " " << grpc_stats_counter_name[i]
         << "/iter:" << ((double)stats.counters[i] / (double)state.iterations());
   }
+  char key[540];
   for (int i = 0; i < GRPC_STATS_HISTOGRAM_COUNT; i++) {
-    out << " " << grpc_stats_histogram_name[i] << "-median:"
-        << grpc_stats_histo_percentile(&stats, (grpc_stats_histograms)i, 50.0)
-        << " " << grpc_stats_histogram_name[i] << "-99p:"
-        << grpc_stats_histo_percentile(&stats, (grpc_stats_histograms)i, 99.0);
+    key[0] = 0;
```

Comment:
Would it be better to use `std::ostringstream`?

Reply:
std::ostringstream